sklearn/datasets/_kddcup99.py

```diff
@@ -296,15 +323,14 @@ def _fetch_brute_kddcup99(data_home=None,
         joblib.dump(y, targets_path, compress=0)
     elif not available:
         if not download_if_missing:
-            raise IOError("Data not found and `download_if_missing` is False")
+            raise OSError("Data not found and `download_if_missing` is False")
 
     try:
         X, y
     except NameError:
         X = joblib.load(samples_path)
         y = joblib.load(targets_path)
-
-    return Bunch(data=X, target=y)
+    return Bunch(data=X, target=y), feature_names, [target_names]
```

Comment:
Why are these not included in the bunch? That usually has ``feature_names`` as well, right?

Reply:
this is the helper function, not the original function itself.

Reply:
Sure, not a big issue, I just didn't see why these wouldn't be included in the bunch here.